src/main/java/seedu/address/model/module/TutorialClass.java

```diff
@@ -19,16 +23,45 @@ public class TutorialClass {
     public static final String VALIDATION_REGEX = "^[A-Z]\\d{2}$";
 
     public final String value;
+    private final ArrayList<Person> students;
+
+    /**
+     * A constructor for TutorialClass. Creates an empty tutorial class with no students.
+     *
+     * @param name of tutorial to be added
+     */
+    public TutorialClass(String name) {
+        requireAllNonNull(name);
+        checkArgument(isValidTutorialClass(name), MESSAGE_CONSTRAINTS);
+        this.value = name;
+        this.students = new ArrayList<>();
+    }
 
     /**
-     * Constructs an {@code TutorialClass}.
+     * A constructor for TutorialClass. Creates a tutorial with one student, that is specified.
      *
-     * @param tutorialClass A valid tutorialClass.
+     * @param name of tutorial to be added
+     * @param student to be added
      */
-    public TutorialClass(String tutorialClass) {
-        requireNonNull(tutorialClass);
-        checkArgument(isValidTutorialClass(tutorialClass), MESSAGE_CONSTRAINTS);
-        value = tutorialClass;
+    public TutorialClass(String name, Person student) {
+        requireAllNonNull(name);
+        checkArgument(isValidTutorialClass(name), MESSAGE_CONSTRAINTS);
+        this.value = name;
+        this.students = new ArrayList<>();
+        students.add(student);
+    }
```

Comment:
Is there a rationale for creating a TutorialClass with only one student?

Reply:
This was done to make it consistent with my Module implementation, though honestly I dont think it will be used. I can remove it